Sources/MongoSwift/ChangeStream.swift

```diff
@@ -50,6 +50,13 @@ public struct ResumeToken: Codable, Equatable {
     }
 }
 
+// TODO: SWIFT-981: Remove this.
+/// The key we use for storing a change stream's namespace in it's `userInfo`. This allows types
+/// using the decoder e.g. `ChangeStreamEvent` to access the namespace even if it is not present in the raw
+/// document the server returns. Ok to force unwrap as initialization never fails.
+// swiftlint:disable:next force_unwrapping
+internal let changeStreamNamespaceKey = CodingUserInfoKey(rawValue: "namespace")!
```

Comment:
ideally this would have lived on `ChangeStream`, but you can't have stored static properties on generic types 🤔